### DISCUSSION TITLE
wish "clear history" function is implemented in the future release

### DISCUSSION THREAD

**Discussion by ghost** [sumatrapdfreader/sumatrapdf#2558] [General]
I don't find any reason not to. Many SPDF lover would benefit from it and appreciate your hard work even more :)

You sometimes don't want others using same pc to know what you've viewed.
"hide frequently read" option would not be of great use for that purpose. they can unhide them with one click.

or there're people like me who just want to feel refreshed, clearing all the past records, and start reading anew.

thank you.

**Comment by GitHubRulesOK** [comment]
To clear history go to settings options and **uncheck** `Remember opened files` and the history should be instantly be cleared (except for favorites)

***HOWEVER there are many different ways that windows tracks file usage that are nothing to do with SumatraPDF such as prefetch and registry MUI and to remove their history is way outside SumatraPDF scope.***

![image](https://user-images.githubusercontent.com/12754913/167866158-3fdc6bea-5544-460c-84ed-b94f93007bb2.png)![image](https://user-images.githubusercontent.com/12754913/167866540-d926b271-9039-4850-b5eb-3766515831a6.png)
![image](https://user-images.githubusercontent.com/12754913/167867518-cb26ce20-0957-46aa-b37b-5003a6bbe997.png)

**Comment by kjk** [comment]
I've added a way to vote for feature requests. That way I'll know what people want the most and prioritize dev work based on that.

https://sumatrapdf.canny.io/feature-requests
